src/ORM/RulesChecker.php

```diff
@@ -298,8 +298,10 @@ public function checkDelete(EntityInterface $entity, array $options = [])
      * @param string $message The error message to show in case the rule does not pass.
      * @return callable
      */
-    public function isUnique(array $fields, $message = 'This value is already in use')
+    public function isUnique(array $fields, $message = null)
     {
+        $message = $message?: __d('cake', 'This value is already in use');
```

Comment:
I think this needs to be:

```php
if (!$message && function_exists('__d')) {
  $message = __d('cake', 'This value is already in use');
}
```
We want to avoid introducing a hard dependency on i18n in the ORM code.